charts/internal/machineclass/templates/machineclass.yaml

```diff
@@ -37,6 +37,9 @@ providerSpec:
     availabilityZone: {{ $machineClass.availabilityZone }}
     flavorName: {{ $machineClass.machineType }}
     keyName: {{ $machineClass.keyName }}
+{{- if $machineClass.subnetID }}
```

Comment:
Is that check now still required when we always pass a subnet id?

Reply:
Nice catch